include/vrt.h

```diff
@@ -58,6 +58,8 @@
  * binary/load-time compatible, increment MAJOR version
  *
  * NEXT (2024-03-15)
+ *	struct vrt_backend.backend_wait_timeout added
```

Comment:
The commit does not actually bump the API version, but adds VRT changes.

Reply:
Because we normally bump as part of the release process.

Reply:
Exactly. I was referring to the commit message [Add vrt ABI bump](https://github.com/varnishcache/varnish-cache/pull/4030/commits/88f83c44a4e5af6d790bddf0feabf15efd9ad7ef)

Reply:
Oh, I believe it is because a bump actually happened in the downstream commit that was cherry-picked here.

@walid-git can you please change the commit message to something like this instead?

> vrt: Register VBE connection queue changes

Reply:
Done